Documentation/building/crossgen.md

```diff
@@ -0,0 +1,64 @@
+Using CrossGen to Create Native Images
+======================================
+
+Introduction
+------------
+
+When you create a .NET assembly using C# compiler, your assembly contains only MSIL code.
+When the app runs, the JIT compiler translates the MSIL code into native code, before the CPU can execute them.
+This execution model has some advantages. For example, your assembly code can be portable across all platforms and architectures that supports .NET Core.
+However, this portability comes with a performance penalty. Your app starts up slower, because the JIT compiler has to spend time to translate the code.
+
+To help make your app start up faster, CoreCLR includes a tool called crossgen, which can pre-compile the MSIL code into native code.
+
+Getting crossgen
```

Comment:
Other than the filename, casing should be consistent. As used above, assuming "CrossGen" is the preferred spelling.